client-default/src/test/scala/com.gu.contentapi.client/GuardianContentClientTest.scala

```diff
@@ -71,11 +71,12 @@ class GuardianContentClientTest extends FlatSpec with Matchers with ScalaFutures
     content.futureValue.id should be (TestItemPath)
   }
 
-  it should "perform a given removed content query" in {
```

Comment:
This seems to be an unrelated change. Was this an irrelevant/unuseful test?

Reply:
That's related to the `I've temporarily made that just check for a 404 to move this along` comment in the PR body :) It'll disappear once I've merged from master now that the 15.6 CAPI models changes are in there.